Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -324,6 +325,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
             SentryIDSetting(settings: self, settingsDelegate: self),
             FasterInactiveTabs(settings: self, settingsDelegate: self),
             OpenFiftyTabsDebugOption(settings: self),
+            FirefoxSuggestSettings(settings: self, settingsDelegate: self)
```

Comment:
nit, we could insert the new debug setting in alphabetical order of the text it shows. In this case it would be after `DeleteExportedDataSetting` and before `ForceCrashSetting`